src/ResourceManager/Automation/Commands.Automation.Test/Commands.Automation.Test.csproj

```diff
@@ -83,8 +87,20 @@
     <Reference Include="Microsoft.IdentityModel.Clients.ActiveDirectory.WindowsForms, Version=2.28.3.860, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <HintPath>..\..\..\packages\Microsoft.IdentityModel.Clients.ActiveDirectory.2.28.3\lib\net45\Microsoft.IdentityModel.Clients.ActiveDirectory.WindowsForms.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Rest.ClientRuntime, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
@ikanni The client runtime version is now coming from a common dependeices target, and has been updated across the board.  Can you pull the latest and resolve any conflicts, in favor of taking the version from common dependencies?

Reply:
Since I believe you just moved to AutoRest, here is an example: https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Compute/Commands.Compute.Test/Commands.Compute.Test.csproj#L408

Reply:
I will fix this.